app/scrapers/stackoverflow.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import print_function
+from generalized import Scraper
+
+
+class StackOverFlow(Scraper):
```

Comment:
Let's use `StackOverflow` instead of `StackOverFlow`.

Reply:
Sure @AnshulMalik